student.rb

```diff
@@ -0,0 +1,16 @@
+require_relative 'person'
+
+class Student < Person
+  def initialize(classroom)
+    @classroom = classroom
+    super(
+      @age = age,
+      @name = name,
+      @parent_permission = parent_permission
+  )
```

Comment:
- Let's also update the super like this below `super(age, name, parent_permission)` then making only the classroom an instance variable like this `@classroom = classroom`